syntax = "proto3";

package fixbug;

option cc_generic_services = true;

import "common.proto";


message InsertFriendRequest
{
    uint32 userid = 1;
    uint32 friendid = 2;
}


message InsertFriendResponse
{
    ResultCode result = 1;
}

message queryFriendsRequest
{
    uint32 userid = 1;
}

message queryFriendsResponse
{
    ResultCode result = 1;
    repeated User friends = 2;
}
// 聊天模块
service friend_model
{
    rpc insert(InsertFriendRequest) returns (InsertFriendResponse);
    rpc query(queryFriendsRequest) returns (queryFriendsResponse);
}